spread.yaml

```diff
@@ -67,16 +67,10 @@ backends:
     type: adhoc
     allocate: |
       multipass_count=$(flock -x .spread_multipass -c 'COUNT=$(cat .spread_multipass); echo $(($COUNT + 1)) | tee .spread_multipass')
-      if [ "$SPREAD_SYSTEM" = "ubuntu-24.04-64" ]; then
```

Comment:
How about keep this and change it to 26.04?

Reply:
Because the Multipass team is working on an improvement this cycle that will make Multipass act like LXD; it will parse simplestreams rather than hardcoding the available buildd images.